packages/phoenix-event-display/src/managers/three-manager/index.ts

```diff
@@ -255,6 +263,16 @@ export class ThreeManager {
     this.clipPlanes[1].normal
       .set(0, 1, 0)
       .applyQuaternion(openingAngleQuaternion);
+    const geometries = this.sceneManager.getGeometries();
```

Comment:
One suggestion for improvement is to set and update the `clipIntersection` conditionally like this:

```ts
const isClipIntersectionInvalid =
  (this.clipIntersection && openingAngle > 180) ||
  (!this.clipIntersection && openingAngle < 180);

if (isClipIntersectionInvalid) {
  this.clipIntersection = openingAngle < 180;
  this.sceneManager.getGeometries().traverse((child) => {
    if (child instanceof Mesh) {
      if (child.material instanceof Material) {
        child.material.clipIntersection = this.clipIntersection;
      }
    }
  });
}
```

This only changes the `clipIntersection` once whenever the user goes below or above 180 degrees and doesn't reach the code every time we change the value. The assignment to `this.clipIntersection` at the top of the function is not needed then.

Reply:
Indeed, that's better than my first implementation. I'll change it.